src/pages/Home.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from 'react';
+import '../App.css';
+import NewListButton from '../components/NewListButton';
+import * as ls from 'local-storage';
+/*import FirebaseTesting from './FirebaseTest';*/
+import fb from '../lib/firebase';
+import { Link, Redirect } from 'react-router-dom';
+import FooterTabs from '../components/FooterTabs';
+/*import ShoppingList from './ShoppingList';
```

Comment:
What are your thoughts on cleaning up some of these commented out lines of code?